recipes/seq/all/conanfile.py

```diff
@@ -0,0 +1,79 @@
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.files import apply_conandata_patches, copy, export_conandata_patches, get, rmdir, rm
+import os
+
+required_conan_version = ">=2.1"
+
+class PackageConan(ConanFile):
+    name = "seq"
+    description = "A collection of original C++14 STL-like containers and related tools"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/Thermadiag/seq"
+    topics = ("formatting", "hashmap", "radix")
+    package_type = "shared-library"
+    settings = "os", "arch", "compiler", "build_type"
+    short_paths = True
+    options = {
+        "header_only": [True, False],
+    }
+    default_options = {
+        "header_only": False,
+    }
+
+    def export_sources(self):
+        export_conandata_patches(self)
+
+    def config_options(self):
```

Comment:
```suggestion
    def configure(self):
```

The `config_options` is too early, it executes before evaluating any option value. The `configure()` has option values computed already.